zeppelin-zengine/src/main/resources/helium/webpack.config.js

```diff
@@ -43,6 +43,8 @@ module.exports = {
       { test: /\.eot(\?\S*)?$/, loader: 'url-loader', }, {
         test: /\.ttf(\?\S*)?$/, loader: 'url-loader', }, {
         test: /\.svg(\?\S*)?$/, loader: 'url-loader', }, {
+        test: /\.png(\?\S*)?$/, loader: 'url-loader', }, {
+        test: /\.jpg(\?\S*)?$/, loader: 'url-loader', }, {
```

Comment:
It would be better to use `webpack.config.js` provided by helium developer. 

- https://issues.apache.org/jira/browse/ZEPPELIN-2192